cli/internal/terraform/loader_test.go

```diff
@@ -186,31 +197,44 @@ func TestPrepareUpgradeWorkspace(t *testing.T) {
 
 			path := path.Join(tc.pathBase, strings.ToLower(tc.provider.String()))
 
-			createFiles(t, file, tc.oldWorkspaceFiles, tc.oldWorkingDir)
-			createFiles(t, file, tc.newWorkspaceFiles, tc.newWorkingDir)
+			createFiles(t, file, tc.workspaceFiles, tc.workingDir)
 
-			err := prepareUpgradeWorkspace(path, file, tc.oldWorkingDir, tc.newWorkingDir, tc.backupDir)
+			err := prepareUpgradeWorkspace(path, file, tc.workingDir, tc.backupDir)
 
 			if tc.wantErr {
 				require.Error(err)
 			} else {
 				require.NoError(err)
+				checkFiles(
+					t, file,
+					func(err error) { assert.NoError(err) },
+					func(content []byte) { assert.NotEqual(defaultFileContent, content) },
```

Comment:
nit: can we rename to 'oldStateContent' or something similar to indicate that this is the current TF file state that should be updated?